[2025-02-02T16:09:14.171+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-02-02T16:09:14.179+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_hooks.pred_time_series manual__2025-02-02T16:08:48.446440+00:00 [queued]>
[2025-02-02T16:09:14.182+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_hooks.pred_time_series manual__2025-02-02T16:08:48.446440+00:00 [queued]>
[2025-02-02T16:09:14.182+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-02-02T16:09:14.186+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): pred_time_series> on 2025-02-02 16:08:48.446440+00:00
[2025-02-02T16:09:14.191+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=450) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-02T16:09:14.192+0000] {standard_task_runner.py:64} INFO - Started process 457 to run task
[2025-02-02T16:09:14.192+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_hooks', 'pred_time_series', 'manual__2025-02-02T16:08:48.446440+00:00', '--job-id', '1485', '--raw', '--subdir', 'DAGS_FOLDER/Stock_pred.py', '--cfg-path', '/tmp/tmpe1vvvhy4']
[2025-02-02T16:09:14.193+0000] {standard_task_runner.py:91} INFO - Job 1485: Subtask pred_time_series
[2025-02-02T16:09:14.209+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_postgres_hooks.pred_time_series manual__2025-02-02T16:08:48.446440+00:00 [running]> on host b7aba691e1ff
[2025-02-02T16:09:14.240+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_postgres_hooks' AIRFLOW_CTX_TASK_ID='pred_time_series' AIRFLOW_CTX_EXECUTION_DATE='2025-02-02T16:08:48.446440+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-02T16:08:48.446440+00:00'
[2025-02-02T16:09:14.241+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-02-02T16:09:14.323+0000] {warnings.py:112} WARNING - /opt/***/dags/Stock_pred.py:275: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_new_ts.rename(index={0:date},inplace=True)

[2025-02-02T16:09:14.324+0000] {logging_mixin.py:188} INFO - Index([  2022-12-30,   2023-01-03,   2023-01-04,   2023-01-05,   2023-01-06,
         2023-01-09,   2023-01-10,   2023-01-19,   2023-01-27,   2023-01-31,
       ...
         2025-01-10,   2025-01-13,   2025-01-14,   2025-01-15,   2025-01-20,
         2025-01-21,   2025-01-22,   2025-01-23,   2025-02-02, '02/02/2025'],
      dtype='object', length=189)
[2025-02-02T16:09:14.334+0000] {base.py:84} INFO - Using connection ID 'postgres_server_2' for task execution.
[2025-02-02T16:09:14.380+0000] {logging_mixin.py:188} INFO -    predicted_index
0       -89.878242
[2025-02-02T16:09:14.386+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-02-02T16:09:14.386+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-02-02T16:09:14.389+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_hooks, task_id=pred_time_series, run_id=manual__2025-02-02T16:08:48.446440+00:00, execution_date=20250202T160848, start_date=20250202T160914, end_date=20250202T160914
[2025-02-02T16:09:14.412+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-02-02T16:09:14.422+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-02T16:09:14.423+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
